scripts/ui/backup_menu.sh

```diff
@@ -17,15 +17,18 @@ function backup_ui() {
   hr
   echo -e "| ${yellow}INFO: Backups are located in '~/kiauh-backups'${white}        |"
   hr
-  echo -e "| Klipper & API:             | Touchscreen GUI:         |"
-  echo -e "|  1) [Klipper]              |  7) [KlipperScreen]      |"
+  echo -e "| Klipper & API:             | Spool Manager:           |"
+  echo -e "|  1) [Klipper]              |  8) [Spoolman]           |"
```

Comment:
I think i would much rather prefer it to be listed under "Other". Same applies to the install, update and remove menu.